platform/vs/docker-sonic-vs/hwsku.json

```diff
@@ -0,0 +1,100 @@
+{
+    "interfaces": {
+        "Ethernet0": {
+	    "default_brkout_mode": "1x100G[40G]"
```

Comment:
>	     [](start = 0, length = 5)

Not well formatted. Mixing tab with spaces

Reply:
Addressed in 3rd commit.